tpp-app/src/home/components/Selector.js

```diff
@@ -45,31 +45,51 @@ const Selector = (props) => {
     let selectedColor = "#B31F20";
     let unselectedColor = "#6D6E71";
 
+    const iconData = [
+        {
+            onPress: () => props.toggleSelectedView(VIEWS.Flow),
+            touchableStyle: [flowSelected && styles.selectedIcon, styles.iconContainer],
+            internalIcon: (<FlowIcon style={styles.icon} fill= {flowSelected ? selectedColor : unselectedColor}/>)
+        },
```

Comment:
```suggestion
        {
            view: VIEWS.Flow,
            selected: flowSelected,
            internalIcon: FlowIcon
        },
```
Repeat this for the rest of the icons in `iconData`